<?xml version="1.0"?>
<launch>

<include file="$(find rover_sim)/launch/default.launch"/>
    

<node pkg="rover_vel_control" type="rover_vel_node" name="rover_vel_node" output="screen">
        <remap from="left_back" to="/rover/corner_lb_wheel_lb_controller/command" />
        <remap from="left_middle" to="/rover/bogie_left_wheel_lm_controller/command" />
        <remap from="left_front" to="/rover/corner_lf_wheel_lf_controller/command" />
        <remap from="right_back" to="/rover/corner_rb_wheel_rb_controller/command" />
        <remap from="right_middle" to="/rover/bogie_right_wheel_rm_controller/command" />
        <remap from="right_front" to="/rover/corner_rf_wheel_rf_controller/command" />
        <remap from="joint_left_front" to="/rover/bogie_left_corner_lf_controller/command" />
        <remap from="joint_left_back" to="/rover/rocker_left_corner_lb_controller/command" />
        <remap from="joint_right_front" to="/rover/bogie_right_corner_rf_controller/command" />
        <remap from="joint_right_back" to="/rover/rocker_right_corner_rb_controller/command" />

</node>
</launch>